tests/System/rms_scripts.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
```

Comment:
I suggest running shellcheck on this and doing what it suggests.

Reply:
Done.